docs/articles/nunit/release-notes/framework.md

```diff
@@ -51,6 +54,17 @@ None
 
 ### Others
 
+* [4765](https://github.com/nunit/nunit/issues/4765) Document ThrowOnEachFailurUnderDebugger. 
```

Comment:
I'm going to also push a "suggestion" to remove the single character of whitespace here to fix the markdown linting

```suggestion
* [4765](https://github.com/nunit/nunit/issues/4765) Document ThrowOnEachFailurUnderDebugger.
```